### DISCUSSION TITLE
Problems with initial install with i2022117

### DISCUSSION THREAD

**Discussion by ceejayemm** [rern/rAudio#1155] [Fixed Issues]
Having had to flash some new SD cards for various issues recently I have found that various settings in the base install (currently i20221117) have a few minor issues:

**System / Environment / Sound profile**
Set to 
vm.swappiness 40
lan mtu 1500
lan txqueuelen 1000

sysctl icon help shows:
# sysctl vm.swappiness
# ifconfig end0 | grep -E 'mtu|txq'

vm.swappiness = 60

Clicking Cog icon shows: No settings
Settings not retained

**Networks** 
Wifi settings show: no settings (see picture 1)
Wifi info icon shows: correct settings (see picture 2)

**Features / Others /AutoPlay**
Enabled Bluetooth Connected to 'On' - settings not retained, function not working as a result.

Chris

Picture 1: ![picture1](https://user-images.githubusercontent.com/25418025/208969235-6f9010d3-9712-4f66-be9c-76ff56462dd1.png)

Picture 2: ![picture2](https://user-images.githubusercontent.com/25418025/208969330-5f011b6b-cd7d-4c01-a08e-944978984640.png)

**Comment by ceejayemm** [comment]
Updating to 202212291 does not fix any of these problems.

Chris

**Reply by ceejayemm** [reply]
Updated to 20230117

WiFi status is now displayed but all other issues still exist.

Chris

**Comment by cmh714** [comment]
Todays update 20230117 has completely hosed up my system....

**Comment by ceejayemm** [comment]
Mine is completely usable other than the issues noted above.  I used the base i20221117 and then applied the released updates up to and including today's but I HAVEN'T updated via pacman on any occasion.

Chris

**Reply by cmh714** [reply]
Completely Fair!! and I am just reporting for the sake of the group :) Luckily I made a full image backup yesterday so quick to fix. My UI is completely fubar'ed, my auto-start playlist is not playing but my Preamp switched over to the USB so its sort of there. Ran a pacman to see if that would magically fix me and nope.

Now to try again....will report back....

**Reply by cmh714** [reply]
Restored my image, reran the Update, and no problems the second time around. Always good to have multiple sd-cards and different backups :)

**Comment by ceejayemm** [comment]
@rern 
I now realise that the Sound Profile options are not currently meant to work with WiFi connections. Why is this ?  Can it be changed ?

I have previously confirmed that the WiFi Status issue is fixed.

That just leaves the problem with:

Features / Others /AutoPlay
Enabled Bluetooth Connected to 'On' - settings not retained, function not working as a result.

I have flashed a new SD card to i20230122 and subsequently updated to 20230123 but this issue still persists.  Is there any chance of it being fixed soon as it greatly improved the usability of rAudio-1 with BT speakers ?  The following output from a recent 'journalctl -xe' command might be useful:

Jan 25 09:30:26 rAudio-1 mpd[8793]: output: Failed to open mixer for 'SoundCore 2 - A2DP': no such mixer control: PCM
Jan 25 09:30:26 rAudio-1 mpd[8793]: D: bluealsa-pcm.c:356: /org/bluealsa/hci0/dev_08_EB_ED_7F_C8_B7/a2dpsrc/sink: Starting
Jan 25 09:30:26 rAudio-1 bluealsa[4401]: /usr/bin/bluealsa: D: ba-transport.c:1409: PCM resumed: 14
Jan 25 09:30:26 rAudio-1 mpd[8793]: D: bluealsa-pcm.c:226: /org/bluealsa/hci0/dev_08_EB_ED_7F_C8_B7/a2dpsrc/sink: Starting IO loop: 25
Jan 25 09:30:26 rAudio-1 audit[9180]: USER_END pid=9180 uid=33 auid=4294967295 ses=4294967295 msg='op=PAM:session_close grantors=pam_limit>
Jan 25 09:30:26 rAudio-1 audit[9180]: CRED_DISP pid=9180 uid=33 auid=4294967295 ses=4294967295 msg='op=PAM:setcred grantors=pam_faillock,p>
Jan 25 09:30:26 rAudio-1 sudo[9180]: pam_unix(sudo:session): session closed for user root

>>>> This might be the issue ?
Jan 25 09:30:26 rAudio-1 mpdidle.sh[9365]: /srv/http/bash/common.sh: line 46: echo: write error: Broken pipe
Jan 25 09:30:26 rAudio-1 mpdidle.sh[9365]: /srv/http/bash/status.sh: line 14: echo: write error: Broken pipe

>>>>
Jan 25 09:30:26 rAudio-1 audit[9545]: USER_ACCT pid=9545 uid=33 auid=4294967295 ses=4294967295 msg='op=PAM:accounting grantors=pam_unix,pa>
Jan 25 09:30:26 rAudio-1 sudo[9545]:     http : PWD=/srv/http ; USER=root ; COMMAND=/srv/http/bash/status.sh undefined
Jan 25 09:30:26 rAudio-1 audit[9545]: CRED_REFR pid=9545 uid=33 auid=4294967295 ses=4294967295 msg='op=PAM:setcred grantors=pam_faillock,p>
Jan 25 09:30:26 rAudio-1 sudo[9545]: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=33)
Jan 25 09:30:26 rAudio-1 audit[9545]: USER_START pid=9545 uid=33 auid=4294967295 ses=4294967295 msg='op=PAM:session_open grantors=pam_limi>
Jan 25 09:30:27 rAudio-1 sudo[9545]: pam_unix(sudo:session): session closed for user root
Jan 25 09:30:27 rAudio-1 audit[9545]: USER_END pid=9545 uid=33 auid=4294967295 ses=4294967295 msg='op=PAM:session_close grantors=pam_limit>
Jan 25 09:30:27 rAudio-1 audit[9545]: CRED_DISP pid=9545 uid=33 auid=4294967295 ses=4294967295 msg='op=PAM:setcred grantors=pam_faillock,p>
Jan 25 09:30:35 rAudio-1 systemd[1]: systemd-timedated.service: Deactivated successfully.

Thanks 

Chris

**Comment by ceejayemm** [comment]
@rern 
Updated to 20230129, no broken pipe now but it is still not possible to enable Features / Others / AutoPlay / Bluetooth Connected.  When I change it to enabled rAudio-1 tells me the Change is happening but on completion Bluetooth Connected is still Disabled.  Clicking on the rAudio-1 logo in the top left hand corner of the screen still shows:

{
	"autoplay": true,
	"autoplayconf": [
		false,
		false,
		true
	],
	"camilladsp": false,


Chris

**Comment by ceejayemm** [comment]
@rern 
Updating to 20230202 now fixes the problem with retaining the enable Features / Others / AutoPlay / Bluetooth Connected setting.  I can see that the BT speaker is recognised when it is powered on BUT there is no audio output.  A message 'Mixer Not Ready Power Off>On / Reconnect' is displayed. I have to go the Networks and Connect the BT device there in order to get sound output.  Turning the BT speaker off is recognised and the sound output is turned off.

Nice try but quite there yet - sorry.

Regards

Chris
